Bottom: 21b89b4719f5438f3351e1caa66b34ed7b489122
Top:    2e8d8dc5b4fb33d80ddf6922ededde937fa7abcf
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2012-11-22 02:54:44 +0000

Refresh of other-drawing-api-changes

---

diff --git a/src/draw.c b/src/draw.c
index bdfdf2a..a5d8b08 100644
--- a/src/draw.c
+++ b/src/draw.c
@@ -93,7 +93,7 @@ set_object_color (AnyObjectType *obj,
                   char *warn_color, char *selected_color,
                   char *found_color, char *normal_color)
 {
-  char *color = PCB->PinColor;
+  char *color;
 
   if      (warn_color     != NULL && TEST_FLAG (WARNFLAG,     obj)) color = warn_color;
   else if (selected_color != NULL && TEST_FLAG (SELECTEDFLAG, obj)) color = selected_color;
